src/gui/SearchDialog.hpp

```diff
@@ -310,6 +315,7 @@ private slots:
 	class SimbadSearcher* simbadSearcher;
 	class SimbadLookupReply* simbadReply;
 	QMap<QString, Vec3d> simbadResults; //! Simbad object name and J2000.0 coordinates
+	QMap<QString, QString> simbadObjectTypes; //! Simbad object name and object type
```

Comment:
Why does this have to be kept as a data member? The only time it's used is when it's just been written, never re-used anywhere else.